indico/htdocs/js/indico/modules/papers.js

```diff
@@ -31,10 +31,35 @@
 
         setupTableSorter('#assignment-list .tablesorter');
         enableIfChecked('#assignment-list', 'input[name=contribution_id]', '.js-enable-if-checked');
-        $('#assignment-list').on('indico:htmlUpdated', function() {
-            setupTableSorter('#assignment-list .tablesorter');
-            applySearchFilters();
-        });
         setupListGenerator(filterConfig);
     };
+
+    global.setupReviewingAreaList = function setupReviewingAreaList(options) {
+        options = $.extend({
+            hasPapers: false
+        }, options);
+
+        if (options.hasPapers) {
+            var filterConfig = {
+                itemHandle: 'div.contribution-row'
+            };
+            if (options.list === 'to-review') {
+                filterConfig = $.extend({
+                    listItems: '#to-review-list div.contribution-row',
+                    term: '#search-input-to-review',
+                    state: '#filtering-state-to-review',
+                    placeholder: '#filter-placeholder-to-review'
```

Comment:
Why not use `'#filter-placeholder-{0}'.format(options.list)` etc? Then you wouldn't have to repeat so much. OTOH, like this the IDs area easily searchable...